backend/models.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+
+from datetime import datetime
+
+from typing import List
+
+
+class Task(BaseModel):
+    """Class for tasks."""
+
+    id: int
+    title: str
+    completed: bool
+
+
+class Todo(BaseModel):
+    """Class for todos."""
+
+    id: int
+    created_at: datetime
+    title: str
+    color: str
+    task: List[Task]
```

Comment:
We didn't discuss what python version we will use, but we should use 3.10 or above.

If that's the case, you can use the following:

```python
task: list[Task]
```

In general, python 3.10 has better support for type hints.